app/client/cluster_api_client.py

```diff
@@ -99,6 +150,38 @@ def is_build_finished():
 
         poll.wait_for(is_build_finished, timeout_seconds=timeout)
 
+    def get_slaves(self):
+        slave_url = self._api.url('slave')
```

Comment:
Added docs.  This is currently only used for testing but I'd like to add a client command to list slaves later.